src/web/pages.rkt

```diff
@@ -96,13 +101,21 @@
     (for/list ([point points])
       (for/list ([value point]) 
         (real->ordinal value repr))))
-
+        
   (define vars (test-vars test))
   (define bits (representation-total-bits repr))
   (define start-error (map ulps->bits-tenths start-errors))
-  (define target-error (and target-errors (map ulps->bits-tenths target-errors)))
+  (define target-error (and target-errors (map (lambda (alt-error) (map ulps->bits-tenths alt-error)) target-errors)))
   (define end-error (map ulps->bits-tenths (car end-errors)))
 
+  (define target-error-entries
+    (if (not target-error)
+        (cons "target" #f)
```

Comment:
Why do we need the #f entry?

Reply:
We needed it for the if case in `report.js` but now with the new code we could remove it